swagger.json

```diff
@@ -2574,9 +2574,234 @@
+                            }
+                        }
+                    },
+                    "default": {
+                        "description": "Unexpected error.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            },
+            "post": {
+                "tags": [
+                    "Default", "Fleet"
+                ],
+                "summary": "/addresses",
+                "description": "Add one or more addresses to the organization",
+                "operationId": "AddOrganizationAddresses",
+                "parameters": [
+                    { "$ref": "#/parameters/accessTokenParam" },
+                    { "$ref": "#/parameters/addressAddParam" }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Addresses were successfully added. No response body is returned."
+                    },
+                    "default": {
+                        "description": "Unexpected error.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/addresses/{addressId}": {
+            "get": {
+                "tags": [
+                    "Default", "Fleet"
+                ],
+                "summary": "/addresses/{addressId}",
+                "description": "Fetch an address by its id",
+                "operationId": "GetOrganizationAddress",
+                "parameters": [
+                    { "$ref": "#/parameters/accessTokenParam" },
+                    {
+                        "name": "addressId",
+                        "in": "path",
+                        "required": true,
+                        "description": "ID of the address/geofence",
+                        "type": "integer",
+                        "format": "int64"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The address/geofence. The geofence field will only be populated with either 'circle' or 'polygon', depending on the geofence type.",
+                        "schema": {
+                            "$ref": "#/definitions/Address"
+                        }
+                    },
+                    "default": {
+                        "description": "Unexpected error.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            },
+            "patch": {
+                "tags": [
+                    "Default", "Fleet"
+                ],
+                "summary": "/addresses/{addressId}",
+                "description": "Update the values of an address",
```

Comment:
`values` is a bit ambiguous. We should list out the fields that can be updated.

Reply:
Could we list out the fields that are eligible to be patched?

Reply:
Sure, should that just take the form of a comma separated list in the description? It doesn't look like there is an option to use markdown or any other type of formatted list.